scripts/initiate_ritual.py

```diff
@@ -111,10 +111,10 @@ def cli(
     access_controller_contract = registry.get_contract(
         domain=domain, contract_name=access_controller
     )
-    fee_model_contract = registry.get_contract(domain=domain, contract_name=fee_model)
+    fee_model_contract = Contract(fee_model)
 
-    # if using a subcription, duration needs to be calculated
-    if fee_model == "StandardSubscription":
+    # if using a subscription, duration needs to be calculated
+    if fee_model_contract.contract_type.name == "StandardSubscription":
```

Comment:
Are you sure this will work? My understanding is that in L114, ape will instantiate `fee_model_contract` as a generic `Contract`, so I'm unsure that `fee_model_contract.contract_type.name` will be available for this check.

Reply:
Yep. Tested it in `ape-console` and by running the command using Marlin's subscription contract and then quitting once the time came to sign a tx. You can try it out.

Using `Contract(...)` should automatically fetch the information from the `ape-etherscan` configured explorer, itf it can't get it locally - https://github.com/ApeWorX/ape/blob/v0.8.14/src/ape/managers/chain.py#L1339.

This has been further improved in newer Ape versions as well - https://docs.apeworx.io/ape/stable/userguides/contracts.html#from-any-address (not our current version, but shows that the pattern holds moving forward).